modules/tensor_mechanics/src/actions/TensorMechanicsActionBase.C

```diff
@@ -230,7 +230,7 @@ TensorMechanicsActionBase::outputPropertiesType()
     for (auto & r : r2cc.second.second)
       options += " " + r2cc.first + "_" + r;
 
-  return MultiMooseEnum(options);
+  return MultiMooseEnum(options, "", true);
```

Comment:
permit _out of range_ options here so that users can supply arbitrary property names